install_config/adding_hosts_to_existing_cluster.adoc

```diff
@@ -250,3 +250,48 @@ infra-node2.example.com openshift_node_labels="{'region': 'infra', 'zone': 'defa
 
 [new_nodes]
 ----
+
+[[adding-etcd-hosts-to-existing-cluster]]
+== Adding etcd Hosts to existing Cluster
+You can add new etcd hosts to your cluster by running the _etcd scaleup_
+playbook. This playbook queries the master, generates and distributes new
+certificates for the new hosts, and then runs the configuration playbooks on the
+new hosts only. Before running the etcd  *_scaleup.yml_* playbook, complete all
+prerequisite
+xref:../install_config/install/host_preparation.adoc#install-config-install-host-preparation[host
+preparation] steps.
+
+To add an etcd host to an existing cluster:
+
+. Ensure you have the latest playbooks by updating the *atomic-openshift-utils* package:
++
+
+[source, bash]
+----
+$ yum update atomic-openshift-utils
+----
+
+. Edit your *_/etc/ansible/hosts_* file, add *new_<host_type>* to the
+*[OSEv3:children]* group and add hosts under the *new_<host_type>* group:
++
+For example, to add a new etcd, add *new_etcd*:
++
+----
+[OSEv3:children]
+masters
+nodes
+[new_etcd]
+new_etcd1.example.com
```

Comment:
`etcd` and `new_etcd` should be added to `[OSEv3:children]` in this example and I'd maybe fill out the etcd group just to show a little more.

```
[OSEv3:children]
masters
nodes
etcd
new_etcd

[etcd]
etcd1.example.com
etcd2.example.com

[new_etcd]
etcd3.example.com
```